extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -107,6 +107,12 @@ public boolean isAnyPropertySet() {
         @ConfigItem(name = ConfigItem.PARENT, defaultValue = "none")
         public String generation = "none";
 
+        /**
+         * Specifies the statement delimiter to be used for the generated DDL scripts.
+         */
+        @ConfigItem(defaultValueDocumentation = "depends on dialect")
```

Comment:
If it depends on the dialect, shouldn't the default value be read from the `Dialect` class used?

Reply:
That sounds like a good idea. I had a quick look at the Dialect class in Hibernate but couldn't find anything corresponding. Which method are you referring to?

Reply:
No idea, just mentioned what is in the proposed `defaultValueDocumentation`. 

Maybe change that to say that the default is `;`?

Reply:
I was thinking it would be more future proof this way. I suspect most developers will want to use the delimiter that makes most sense with their dialect / DDL tool. A hardcoded `;` may mean that all users using some given future dialect may have to override this property. But maybe I just need to add some more details to the default value documentation.

Reply:
I had another look and I couldn't find any SQL client which doesn't use `;` as the default statement terminator. Many clients allow changing the default because semicolons are often used inside stored procedures, triggers, etc. and the client would then have problems disambiguating. But I am inclined to change the default to `;` and remove the `defaultValueDocumentation` again. Any other opinions regarding this?

Reply:
That sounds good